lib/kubernetes-deploy/runner.rb

```diff
@@ -319,7 +319,7 @@ def phase_heading(phase_name)
     end
 
     def log_green(msg)
-      STDOUT.puts "\033[0;32m#{msg}\x1b[0m\n" # green
+      KubernetesDeploy.logger.info("\033[0;32m#{msg}\x1b[0m")
```

Comment:
Added this to the master issue under "code" (this change is actually from the branch this is based on, which I already merged, and will disappear when I push the rebase).